lib/connextclient/ConnextClient.ts

```diff
@@ -692,6 +694,44 @@ class ConnextClient extends SwapClient {
     });
   }
 
+  public async withdraw({
+    all,
+    currency,
+    amount: argAmount,
+    destination,
+    fee,
+  }: WithdrawArguments): Promise<string> {
+    if (currency === 'ETH') {
+      // TODO: query Ether balance, subtract gas price times 21000 (gas usage of transferring Ether), and set that as amount
+      throw Error('Withdrawing all ETH is not supported yet');
```

Comment:
Should this if statement only fall under the `if (all)` bracket? As is we wouldn't support any ETH withdrawal.